charts/goblackhole/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: goblackhole
+description: Offical Helm Chart for goblackhole
+type: application
+version: 0.0.1
+appVersion: 0.2.0
+keywords:
+  - bgp
+  - blackhole
+  - network
+home: https://github.com/bedag/goblackhole
+icon: https://raw.githubusercontent.com/bedag/goblackhole/main/media/gopher.png
+sources:
+  - https://github.com/bedag/goblackhole
+  - https://hub.docker.com/r/bedag/goblackhole
+maintainers:
+  - name: NOC
+    email: noc@bedag.ch
+dependencies:
+- name: manifests
+  version: "~0.5.0"
+  repository: https://bedag.github.io/helm-charts
+annotations:
+  artifacthub.io/containsSecurityUpdates: "false"
+  artifacthub.io/prerelease: "false"  
+  artifacthub.io/changes: |
+    - "[Added]: Chart Release"
+  artifacthub.io/images: |
+    - name: goblackhole
+      image: bedag/goblackhole:0.1.0
```

Comment:
I think that's no longer required:

> artifacthub.io/images (yaml string, see example below)
By default, Artifact Hub will try to extract the containers images used by Helm charts from the manifests generated from a dry-run install using the default values. If you prefer, you can also provide a list of containers images manually by using this annotation.

Reply:
a nice thxs :=)